app.json

```diff
@@ -0,0 +1,9 @@
+{
+    "name": "asgard-cfh",
+    "scripts": {},
```

Comment:
Everything here except for `name` seems redundant as they hold no actual value, please get rid of them ASAP. Good job @clintfidel 😄

Reply:
Also, link to the hosted app isn't correct. Please fix it.